playbooks/globalVariables.json

```diff
@@ -0,0 +1,12 @@
+[
+    {
+        "name": "Server_fqhn",
+        "value": "localhost.localdomain",
+        "default_value": "localhost.localdomain"
```

Comment:
IMO, this is not required. Please confirm

Reply:
Not needed, Removed